message/messaging_service.cc

```diff
@@ -482,6 +482,13 @@ future<> messaging_service::stop_client() {
             return c.second.rpc_client->stop().then([addr = c.first] {
                 mlogger.info("Stopping client for address: {} - Done", addr);
             });
+        }).then([&m] {
```

Comment:
If you used `finally()` here, you wouldn't need the first part of the comment below.

Reply:
Good point. Fixed